cinder_service_check.py

```diff
@@ -53,6 +53,12 @@ def check(auth_ref, args):
 
     services = r.json()['services']
 
+    # We need to match against a host of X and X@lvm (or whatever backend)
+    if args.host:
+        backend = ''.join((args.host, '@'))
```

Comment:
Using `str.join` for two items. It's vastly more performent for an indeterminate number of strings, but not when you're using just 2. Doing `args.host + '@'` would be much better in this case.